components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
little typo in "product"
```suggestion
export const Button = ({ setProduct, product, productToBuy }) => {
```